tdrs-backend/Pipfile

```diff
@@ -44,6 +44,7 @@ requests = "~=2.25.0"
 django-admin-logs = "~=1.0.1"
 drf-yasg = "~=1.20.0"
 wait-for-it = "==2.2.0"
+django-admin-508 = "==0.0.2"
```

Comment:
very cool to see [this](https://pypi.org/project/django-admin-508/0.0.2/)@jorgegonzalez!